### PR TITLE
[10.0] [MIG] base custom info

### PR DESCRIPTION
**Pull request by fanha99**
almost change nothing from 9.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis status

**Comment by fanha99** [comment]
hi,
I got this info from the travis:

> FAIL: test_cookie_as_string (odoo.addons.sentry.tests.test_logutils.TestOdooCookieSanitizer)
Traceback (most recent call last):
`   File "/home/travis/build/OCA/server-tools/sentry/tests/test_logutils.py", line 35, in test_cookie_as_string
`     m=proc.MASK,
` AssertionError: 'website_lang=en_us;session_id=hello;Session_ID=hello;foo=bar' != 'website_lang=en_us;session_id=********;Session_ID=********;foo=bar'
FAIL: test_cookie_header (odoo.addons.sentry.tests.test_logutils.TestOdooCookieSanitizer)
Traceback (most recent call last):
`   File "/home/travis/build/OCA/server-tools/sentry/tests/test_logutils.py", line 78, in test_cookie_header
`     'a_session_id_here={m}'.format(m=proc.MASK))
` AssertionError: 'foo=bar;session_id=hello;Session_ID=hello;a_session_id_here=hello' != 'foo=bar;session_id=********;Session_ID=********;a_session_id_here=********'
FAILED
Module sentry: 2 failures, 0 errors

Any Idea how to fix?

**Comment by yajo** [comment]
Wouldn't it be better to leave all as LGPL for an addon such as this one? 🤔

**Comment by pedrobaeza** [comment]
Module is LGPL already, but I think we missed to change manifest key.

**Comment by fanha99** [comment]
Hi all, in this example of readme: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0
There is AGPL. There module in version 9 is LGPL. 
Am I need to switch back to LGPL?
Edit: correct the version number.

**Comment by pedrobaeza** [comment]
I think so

**Comment by fanha99** [comment]
Hi all, switch back to LGPL as v9.
please help to review.

**Comment by fanha99** [comment]
@Yajo : I'm working this module for v11.
I got the same problem: "When you choose a template from the partner form, it fills the values, but properties remain empty" is there any hint to fix it?

**Comment by yajo** [comment]
@fanha99 I remember problems with onchange implementation. It was supposedly fixed in Odoo itself, although in the mean time we managed to get a workaround. Read from https://github.com/OCA/server-tools/pull/492#issuecomment-237594285 onwards, all I can say to you must be there.

OTOH, it's easier to discuss on the v11 PR, with runbot & so on. Let's focus on finishing this one and then you can open the v11 and we can talk on that if you like.

**Review by yajo — APPROVED** [review]

File: base_custom_info/__manifest__.py [hunk 1]
```diff
@@ -2,34 +2,46 @@
...
     ],
-    'author': 'Antiun Ingeniería S.L., '
-              'Incaser Informatica S.L., '
-              'Tecnativa, '
+    'author': 'Tecnativa, '
               'Odoo Community Association (OCA)',
-    'website': 'http://www.antiun.com',
-    'license': 'AGPL-3',
-    'installable': False,
+    'website': 'https://www.tecnativa.com',
```
Comment (leemannd) [inline base_custom_info/__manifest__.py]: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L9

File: base_custom_info/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # © 2015 Antiun Ingeniería S.L. - Sergio Teruel
```
Comment (leemannd) [inline base_custom_info/__init__.py]: all s/©/Copyright/

**Review by leemannd — CHANGES_REQUESTED** [review]
A few nitpicking, but averall work looks good to me.
Thanks!

File: base_custom_info/models/custom_info_property.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+             "options here.",
+    )
+
+    @api.multi
+    def check_access_rule(self, operation):
+        """You access a property if you access its template."""
+        self.mapped("template_id").check_access_rule(operation)
+        return super(CustomInfoProperty, self).check_access_rule(operation)
+
+    @api.one
```
Comment (TelmoSenseFly) [inline base_custom_info/models/custom_info_property.py]: Isn't the deprecation of ```api.one``` after version ```9.0``` in place?
https://www.odoo.com/documentation/10.0/reference/orm.html#module-odoo.api

**Comment by leemannd** [comment]
@TelmoSenseFly Not sure it can be merged now. We have internal testing coming tomorrow.
Do you want the release to be done without it, or shall we wait for tomorrow?

**Comment by TelmoSenseFly** [comment]
@leemannd No worries. It will be fine to wait for tomorrow.

**Comment by fanha99** [comment]
hi @leemannd , @TelmoSenseFly, could you please review again? thx.

**Comment by diggy128** [comment]
Could this be merged?

**Comment by pedrobaeza** [comment]
I think this doesn't contain latest changes in v9.

**Comment by diggy128** [comment]
OK. Since I use this module along with other that depend on it, should I try to migrate it after a rebase?

**Comment by pedrobaeza** [comment]
OK

**Review by leemannd — APPROVED** [review]

**Comment by fanha99** [comment]
> 
> 
> I think this doesn't contain latest changes in v9.

could you please tell me which changes is it?

**Comment by pedrobaeza** [comment]
@fanha99 sorry, they are there. Please squash your 2 migration commits and I'll merge.

**Comment by fanha99** [comment]
@pedrobaeza : done

**Pull request merged**